### ISSUE TITLE
✨Field cloneElement replacement by Injector

### ISSUE THREAD

**Issue by andresin87** [SUI-Components/SUIComponents#2335]
### Describe the solution you'd like

Replace the `cloneElement`in innerLabel function by an `Injector`primitive wrapper of `nodelabel` and `label` cases 

### Additional context**

blocked by #2334

**Comment by oegea** [comment]
I'll work on this if possible :) @andresin87

**Comment by oegea** [comment]
Could this be a duplicate of #2321?

**Comment by andresin87** [comment]
This is only for the `innerLabel ` function of Molecule Field

**Comment by andresin87** [comment]
Take a look this task is blocked by #2321
